apps/zotonic_mod_translation/src/support/translation_translate.erl

```diff
@@ -0,0 +1,154 @@
+%% @author Marc Worrell <marc@worrell.nl>
+%% @copyright 2023 Driebit BV
+%% @doc Translate strings from one language to another. Try to use the .po files and
+%% optional translation services provided by other modules.
+%% @end
+
+%% Copyright 2023 Driebit BV
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(translation_translate).
```

Comment:
Or maybe `translation_service`

Reply:
I didn't call it service, as it is trying to translate, using any means we have.

Reply:
In this case "service" would mean: the internal translation functionality (service) that can translate. Indeed, it could also mean "external service". But I see the point.